test/example/test_inventory.jl

```diff
@@ -0,0 +1,54 @@
+# Assumptions
+#
+# * current working directory is the root directory of the project ("DocInventories") in this case
```

Comment:
You're testing the "example" here, not the "DocInventories" documentation, right?

Reply:
I am...will clean up the comments!